etc/linux-sysctl/README.md

```diff
@@ -0,0 +1,21 @@
+sysctl configuration to raise UDP buffer size
+===================
+Installation
+-----------
+**Please note:** When you installed syncthing using the official deb package, you can skip the copying.
+
+Copy the file `30-syncthing.conf` to `/etc/sysctl.d/` (root permissions required).
+
+In a terminal run
+```
+sudo systemctl restart procps
```

Comment:
Do we need to be this systemd specific here, instead of for example `sysctl -q --system` to load the things directly?

Reply:
Works both. I can change it.